users/service_test.go

```diff
@@ -141,12 +141,12 @@ func TestUserInfo(t *testing.T) {
 	}
 }
 
-// UpdateUser updates the user metadata
-
 func TestUpdateUser(t *testing.T) {
 	svc := newService()
 	svc.Register(context.Background(), user)
 	key, _ := svc.Login(context.Background(), user)
+	nonExistingKey, _ := svc.Login(context.Background(), nonExistingUser)
```

Comment:
you cant use login here, you havent registered user, this will fail
to create key
```
j := jwt.New("secret")
key, _ := j.TemporaryKey(user.Email)
```

Reply:
@mteodor I agree with you but let's just remove this test. No need to create a valid token for this test.